js/adapt-contrib-glossaryItemView.js

```diff
@@ -58,8 +58,10 @@ define(function(require) {
          */
         showGlossaryItemDescription: function() {
             var $glossaryItemTerm = this.$('.glossary-item-term');
-            var description = $glossaryItemTerm.addClass('selected').siblings('.glossary-item-description').slideDown(200, function() {
-                $(description).a11y_focus();
+            var $description = $glossaryItemTerm.addClass('selected').siblings('.glossary-item-description');
+            $description.slideDown(200, function() {
+            	$description.parents('.drawer').scrollTo($glossaryItemTerm);
```

Comment:
It's by no means perfect, but if you do this:
```js
if(this.getBoundingClientRect().top >= document.documentElement.clientHeight) {
    $description.parents('.drawer').scrollTo($glossaryItemTerm);
}
```
it should only scroll the term to the top if the description is off-screen when the term is clicked.

Reply:
@tomgreenfield @chris-steele you can see this working in uk-lcli-405767